seismic-app/src/components/Footer.js

```diff
@@ -0,0 +1,105 @@
+import "../css/footer.css";
+import React from "react";
+import { Row, Col, Typography, Input, Anchor } from "antd";
+
+const { Title, Text } = Typography;
+const { Link } = Anchor;
+
+function subscribe(event) {
+  const email = event.target.value;
+  console.log(`TODO: send email to ${email}`);
+}
+
+const text = {
+  color: "white",
+  textAlign: window.innerWidth > 768 ? "start" : "center"
+};
+
+const Footer = props => {
+  const [width, setWidth] = React.useState(window.innerWidth);
+  const [textStyle, setTextStyle] = React.useState({
+    textAlign: window.innerWidth > 768 ? "start" : "center"
```

Comment:
At a glance it looks like `textStyle` is pretty much a duplicate of `text` (looks like all text is white anyways). But `textStyle` is updated on window resize while `text` is not.
Suggestion: Add `color: white` to `textStyle` (or define the color in css) and replace all `text` usages with `textStyle`. Remove the definition of `text`. Would this make sense?

Reply:
textStyle is used on the parent element, while text is used in typography elements. Color does not apply if used on the parent element, so the only solution i got is to have 2 variables for this, textStyle for parent with align and text for text color. textAlign in text was a mistake

Reply:
> Color does not apply if used on the parent element

That only happens for `<a>` tags or things that override `color`. But that can be over-overridden in CSS. Something like this:

```css
.footer-inside a {
    color: white;
}
```

Reply:
The problem isn't inside <a>, but with every typography tag, both <Title> and <Text>. The only other solution I found was creating a theme, but it would be on the entire project, not just this component

Reply:
Ah, you mean the style is taken over by Ant. Well, there's always the magical `!important` 😃 

```css
.footer-inside a {
    color: white !important;
}
```

Reply:
Didn't work like that either, but i found the solution
```css
.ant-typography {
  color: white !important;
}
```